react/jsx-runtime module not found

I'm getting the following build error. 
Using React: 17. 
Implemented according to guide.

Any idea on how to solve it?

`Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/Users/usr/folder/project/node_modules/react/jsx-runtime' imported from /Users/usr/folder/project/node_modules/@vercel/analytics/dist/react/index.js
Did you mean to import react/jsx-runtime.js?`

--- a/packages/web/src/react.tsx
+++ b/packages/web/src/react.tsx
@@ -1,4 +1,4 @@
-import { useEffect } from 'react';
+import React, { useEffect } from 'react';
 import { inject } from './generic';
 import type { BeforeSend } from './types';
 

--- a/tsconfig.json
+++ b/tsconfig.json
@@ -8,7 +8,7 @@
     "noUnusedLocals": true,
     "skipLibCheck": true,
     "sourceMap": true,
-    "jsx": "react-jsx",
+    "jsx": "react",
     "moduleResolution": "node"
   }
 }

